ecosystem/typescript/sdk/examples/typescript-esm/rotate_key.ts

```diff
@@ -0,0 +1,56 @@
+import { AptosAccount, FaucetClient, Network, Provider, HexString } from 'aptos';
```

Comment:
curios why decided to use examples/typescript-esm and not examples/typescript

Reply:
I wasn't sure which one to use, since I thought you said we would be slowly moving to esm. But I may have misunderstood. Also I kind of just wanted to organize mine in a different folder since I'm probably going to be adding a bunch of examples over time 😅

should I use the other one?

Reply:
ohh no right or wrong decision, was just curios, feel free to use whatever you like. I might revisit this whole examples folder later and think on what to do with it - don't think we need all those different folders.